test/Tester/ClientConnectionTests/ClientDisconnectionEventTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.TestingHost;
+using TestExtensions;
+using UnitTests.GrainInterfaces;
+using Xunit;
+
+namespace Tester.ClientConnectionTests
+{
+    public class ClientDisconnectionEventTests : TestClusterPerTest
+    {
+        public override TestCluster CreateTestCluster()
+        {
+            return new TestCluster(new TestClusterOptions(2));
+        }
+
+        [Fact, TestCategory("BVT")]
+        public async Task EventSendWhenDisconnectedFromCluster()
+        {
+            var runtime = (OutsideRuntimeClient) RuntimeClient.Current;
```

Comment:
I'm trying to get rid of `RuntimeClient.Current`. Could you grab this using DI instead?